reporter/src/main/kotlin/reporters/CycloneDxReporter.kt

```diff
@@ -43,6 +43,7 @@ import org.ossreviewtoolkit.model.utils.toPurl
 import org.ossreviewtoolkit.reporter.Reporter
 import org.ossreviewtoolkit.reporter.ReporterInput
 import org.ossreviewtoolkit.spdx.SpdxLicense
+import org.ossreviewtoolkit.utils.CopyrightStatementsProcessor
```

Comment:
Commit message nit:

> Before the processing

This should be changes to "Before,  the processing" or "Previously, the processing" as the comma changes the meaning of the sentence.

Reply:
Done.